osc/commandline.py

```diff
@@ -7627,7 +7645,21 @@ def do_service(self, subcmd, opts, *args):
             elif command in ("runall", "ra"):
                 mode = "all"
 
-        return p.run_source_services(mode, singleservice, opts.verbose)
+        if store:
+            # we're in a working copy
+            p = Package(".")
+            return p.run_source_services(mode, singleservice, opts.verbose)
+        else:
+            # we're in a git repo
+            si = Serviceinfo()
+            si.apiurl = apiurl
+            si.project = project
+            si.package = package
+
+            if os.path.isfile("_service"):
+                si.read(ET.parse("_service"))
+
+            return si.execute(dir=".", callmode=mode, singleservice=singleservice)
```

Comment:
I think this is pretty hacky .. we need to decide where to store the information. I wonder if checking `.osc/_project` into git wouldn't be the easiest solution (works with feature branches, doesn't need osc hacks). 

we could also make a fallback `.obsconfig/_project` which `Package()` falls back to for reading but not for writing.

Reply:
This could lead to discrepancies between git and obs, unless the a `git/osc clone/pull/update` automatically updates this with the settings from OBS.